src/shared/modules/offer/default-offer.service.ts

```diff
@@ -77,4 +77,39 @@ export class DefaultOfferService implements OfferService {
       .populate(['userId'])
       .exec();
   }
+
+  public async findByPremium(cityName: string): Promise<DocumentType<OfferEntity>[]> {
+    return await this.offerModel.find(
```

Comment:
public async findByPremium(cityName: string): Promise<DocumentType<OfferEntity>[]> => this.offerModel.find(